tests/integration/targets/meraki_mr_l3_firewall/tasks/main.yml

```diff
@@ -17,6 +17,28 @@
   - set_fact:
       net: '{{new_net.data.id}}'
 
+  - name: Check rule values are lowercase
```

Comment:
This test is failing. I'm thinking you need to include a `number` argument specifying which SSID to look up.

```
fatal: [localhost]: FAILED! => {"changed": false, "msg": "HTTP error 400 - https://api.meraki.com/api/v1/networks/N_624874448297691233/wireless/ssids/None/firewall/l3FirewallRules - SSID number must be an integer", "response": "OK (unknown bytes)", "status": 400}
```